wolfssl/ssl.h

```diff
@@ -65,6 +65,16 @@
     extern "C" {
 #endif
 
+#ifndef WC_RNG_TYPE_DEFINED /* guard on redeclaration */
+    #ifdef HAVE_FIPS
+        /* use typedef struct from CTaoCrypt */
+        #include <cyassl/ctaocrypt/random.h>
```

Comment:
ssl.h doesn't currently include any module level headers (TLS, Crypto) just simple defines files settings and version.  This is to keep types opaque. 

Is there a solution that doesn't require changing this pattern?